src/store/sagas/handlers/user.js

```diff
@@ -42,8 +43,9 @@ export function* handleRegisterUser(action) {
 
 export function* handleLoginUser(action) {
     try {
-
+        
         yield call(requestLoginUser.bind(null, action.userData))
+        yield put(enableButton())
```

Comment:
What is to say they don't try clicking the button again before the page loads them into the app after successfully authenticated...

Reply:
I don't get this.

Reply:
if the button gets activated after the process is done, we risk give the user starting another authentication process before the page decides to reload.

Reply:
Hmm, I see. So what do you suggest we do instead.

Reply:
Nothing actually...

Reply:
So we should leave it as it is. So let's say   a user is approved but not yet verified, after logging in they get to their user dashboard. Then the button is disabled which isn't supposed to be so. Ideally on the login page after logging in the button becomes enabled for a split second before redirecting and the button on the dashboard isn't disabled again with my implementation. What about using`history.push` instead of using `Redirect`

Reply:
Let's use `Redirect` for it all. We aren't using `Snapshot` data so a reload is very much needed to get updated database info. no `history.push `... all `redirects`

Reply:
My bad...I did go through my code process again...was trying to understand what you were saying.

And I figured out you were right. the button does stay disabled on redirect.

I can approve this PR it seems like.

Reply:
Thanks for the heads up.